index.html

```diff
@@ -0,0 +1,52 @@
+<link rel="stylesheet" href="css/style.css">
```

Comment:
Great job on explaining why these changes should be made 💪 however I think it would be better to phrase the comment more politely in order to keep the communication with the student more friendly, for example, I think it would be better to make sure... instead of Please make sure...